CorsixTH/Lua/world.lua

```diff
@@ -1082,6 +1082,7 @@ end
 
 -- Called immediately prior to the ingame day changing.
 function World:onEndDay()
```

Comment:
While historically, `world` may have been the central boss, the code that you touch here hasn't much to do with `world`.
Instead, it seems to belong in the hospital class instead.

Reply:
Again the purpose of this PR is not lifting out potentially misplaced actions in ``onEndDay()``